fccodegen/tests/test_printer.py

```diff
@@ -0,0 +1,197 @@
+#
+# Tests conversion of sympy expressions to weblab cython code.
+#
+import fccodegen as cg
+import logging
+import math
+import pytest
+import sympy as sp
+
+
+# Show more logging output
+logging.getLogger().setLevel(logging.INFO)
+
+
+def test_simple():
+    x, y, z = sp.symbols('x y z')
+
+    p = cg.WebLabPrinter()
+
+    # Number types
+    assert p.doprint(1) == '1'                  # int
+    assert p.doprint(1.2) == '1.2'              # float, short format
+    assert p.doprint(math.pi) == '3.141592653589793'    # float, long format
+    assert p.doprint(1.436432635636e-123) == '1.436432635636e-123'
+    assert p.doprint(x - x) == '0'              # Zero
+    assert p.doprint(x / x) == '1'              # One
+    assert p.doprint(-x / x) == '-1'            # Negative one
+    assert p.doprint(5 * (x / x)) == '5'        # Sympy integer
+    assert p.doprint(5.5 * (x / x)) == '5.5'        # Sympy float
+    assert p.doprint(sp.Rational(5, 7)) == '5 / 7'  # Sympy rational
+
+    # Special numbers
+    assert p.doprint(sp.pi) == 'math.pi'
+    assert p.doprint(sp.E) == 'math.e'
+
+    # Symbols
```

Comment:
These things should really be split into multiple test methods. I don't go as far as saying a single `assert` per method, but a test method per category would be good.

Reply:
Done!

Reply:
Nice use of fixtures!